clar2wasm/src/standard/standard.wat

```diff
@@ -1409,29 +1409,81 @@
     )
 
     (func $stdlib.sha256-buf (param $offset i32) (param $length i32) (param $offset-result i32) (result i32 i32)
+        ;; For binary representation, you can take a look at https://sha256algorithm.com/
+        ;; For initial values, constants and high level understanding of SHA-512, please take a look at this Rust implementation
+        ;; https://github.com/dandyvica/sha/blob/master/src/sha256.rs
```

Comment:
That seems very random. Why would we need to go see this implementation for the values? You can find them everywhere. For me I found them in the repo of sha256algorithm.com

Reply:
> For me I found them in the repo of sha256algorithm.com

This is also the implementation, but in Javascript. Developer doesn't have to see the implementation, just grab values and constants and that's it. One can grab those values and constants from the paper. Paper reference updated.

Reply:
Exactly my point, there are bazillions of implementations in all possible languages, what's the point of linking this one?